src/hooks/useValidateRepository.ts

```diff
@@ -65,81 +83,84 @@ const useValidateRepository = (
             usernameSecret: current.usernameSecret,
           };
         }
+
         return {...acc, [name]: current[name as keyof typeof current]};
       }, {}),
     };
   };
 
-  const handleResponse = (res: any) => {
+  const handleResponse = (res: RTKResponse<void>) => {
     if (!isEqual(latestRef.current, current)) {
       return;
     }
 
     // Save information about the last response
     validatedRef.current = current;
 
-    if (res?.error) {
-      const errorDetail = res?.error?.data?.detail;
+    if (res && 'error' in res && 'data' in res.error) {
+      // @ts-ignore
+      const errorDetail = res.error.data?.detail;
```

Comment:
Given how this condition looks like and it still needs `ts-ignore`, probably it's better to cast it:
```suggestion
    const error = (res as any).error;
    if (error) {
      const errorDetail = error.data?.detail;
```